Content.Server/Labels/Label/LabelSystem.cs

```diff
@@ -31,6 +32,39 @@ public override void Initialize()
             SubscribeLocalEvent<PaperLabelComponent, ExaminedEvent>(OnExamined);
         }
 
+        /// <summary>
+        /// Apply or remove a label on an entity.
+        /// </summary>
+        /// <param name="uid">EntityUid to change label on</param>
+        /// <param name="text">intended label text (null to remove)</param>
+        /// <param name="label">label component for resolve</param>
+        /// <param name="metadata">metadata component for resolve</param>
+        public void Label(EntityUid uid, string? text, MetaDataComponent? metadata = null, LabelComponent? label = null)
+        {
+            if (!Resolve(uid, ref metadata))
+                return;
+            if (!Resolve(uid, ref label, false))
+                label = uid.EnsureComponent<LabelComponent>();
```

Comment:
The uid extension methods hide a `IoCManager.Resolve<IEntityManager>()` and generally shouldn't be used (They should probably be marked as obsolete).

Just use `EnsureComp<LabelComponent>(uid)` instead